exonum/src/blockchain/mod.rs

```diff
@@ -343,6 +343,9 @@ impl Blockchain {
                     schema.precommits_mut(&block_hash).push(precommit.clone());
                 }
 
+                // cleanup the consensus messages cache
+                schema.consensus_messages_cache_mut().clear();
```

Comment:
I'm not sure that we need such comments. :smile: The code is pretty self-explanatory:  `....consensus_messages_cache...().clear();`.

In my opinion comments are needed in the following cases:
1. Code is non-trivial.
2. Intent isn't obvious (for example, you can want to explain why we need to clear the cache).